cypress/integration/gallery-spec.js

```diff
@@ -0,0 +1,57 @@
+import { Preview } from '../page-objects/preview-page'
+import { Gallery } from '../page-objects/gallery-page'
+
+const preview = new Preview()
+const gallery = new Gallery()
+
+describe( 'Check the Gallery Pages ', () => {
+
+	beforeEach( 'Open the English Page', () => {
+		cy.navigateToTestPage()
+	} )
+
+	it( 'Check the Gallery Pages', () => {
+		// Open the Preview
+		preview.getPreviewSpan().first().click()
+		// Check for the Continue Reading exist
+		preview.getFooterContiReading().then( res => {
+			if ( res.css( 'display' ) !== 'none' ) {
+				preview.getFooterContiReading().click()
+			}
+		} )
+		// Check if the Images exist
+		preview.getBodyGalleryImages().its( 'length' ).then( res => {
+			if ( res > 0 ) {
```

Comment:
if the res == 0 what is this test doing?

Reply:
This is added to make sure that the images are present in the Gallery. The test is about visiting the gallery mode. Thereby it needs to have at least one image present in the preview content.

Reply:
since u can add any words to the test.html page, why not add articles with images to be sure they're present?
no need to add code checking that the image is there, this test won't run/test anything if u find an article without images

Reply:
Firstly, the images depend on the content they receive from the API. Suppose there is a change in content in the future then our condition takes care of it. 
Secondly, it just ensures that the proper number of images is there for testing. It is a fallback condition and never disrupts the test.
Lastly, I have chosen a word such that it has images and always falls inside the condition.Hence, it will never fail.
Thereby, I believe that this condition is necessary.